docs/howto/distributed-tracing/ref_what-is-distributed-tracing.adoc

```diff
@@ -6,7 +6,7 @@ In the microservices architecture, where multiple services serve a single user r
 One technique that makes debugging easier is called _distributed tracing_.
 Distributed tracing is collecting information about service invocations, correlating it to find all invocations related to a single user request, and visualizing the data in a form that helps you understand the hierarchy of invocations and makes finding problems easier.
 
-{Thorntail} provides a fraction for MicroProfile OpenTracing, an API for distributed tracing, built on JAX-RS and OpenTracing.
+{Thorntail} provides a fraction for Eclipse MicroProfile OpenTracing, an API for distributed tracing, built on JAX-RS and OpenTracing.
```

Comment:
This one is good.

Reply:
Ok, thank you:)

Reply:
This is the one at the beginning of the Distributed Tracing guide.
That justifies removing the ones in the lower sections of the guide.